From: Frank Rowand <frank.rowand@am.sony.com>
Subject: [PATCH 4/4] RT: change from raw_spinlock_t to __raw_spinlock_t
To: linux-kernel@vger.kernel.org
Cc: mingo@redhat.com, tglx@linutronix.de
Date: Tue, 15 Jan 2008 14:21:46 -0800

From: Frank Rowand <frank.rowand@am.sony.com>

Fix compile warning (which becomes compile error due to -Werror),
by changing from raw_spinlock_t to __raw_spinlock_t.

Signed-off-by: Frank Rowand <frank.rowand@am.sony.com>
---
 arch/mips/kernel/gdb-stub.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.26/arch/mips/kernel/gdb-stub.c
===================================================================
--- linux-2.6.26.orig/arch/mips/kernel/gdb-stub.c
+++ linux-2.6.26/arch/mips/kernel/gdb-stub.c
@@ -176,7 +176,7 @@ int kgdb_enabled;
  * spin locks for smp case
  */
 static DEFINE_SPINLOCK(kgdb_lock);
-static raw_spinlock_t kgdb_cpulock[NR_CPUS] = {
+static __raw_spinlock_t kgdb_cpulock[NR_CPUS] = {
 	[0 ... NR_CPUS-1] = __RAW_SPIN_LOCK_UNLOCKED,
 };
 
